features/steps/ruby_notifier_steps.rb

```diff
@@ -22,4 +22,10 @@
   steps %Q{
     the "#{element}" of stack frame #{frame_index} equals #{value}
   }
+end
+
+Then(/^the total sessionStarted count equals (\d+)(?: for request (\d+))?$/) do |value, request_index|
+  session_counts = read_key_path(find_request(request_index)[:body], "sessionCounts")
+  total_count = session_counts.inject(0) { |count, session| count += session["sessionsStarted"] }
+  assert_equal(total_count, value)
```

Comment:
Flip these arguments (`(value, total_count)`); the first parameter is `expected` and the second is `actual`.